src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  padding: 32px 16px 16px;
+  background-color: #fff;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  align-items: center;
+  transition: transform 0.3s ease;
+
+  &:hover {
+    transform: scale(1.2);
+
+    &__details-name {
+      color: #34568b;
+    }
+  }
+
+  &__product {
+    display: flex;
+    &-image {
+      width: 160px;
+      height: 134px;
+    }
+  }
+
+  &__details {
+    margin: 40px 0 0;
+    &-name {
+      width: 166px;
+      font-size: 12px;
+      color: $primary-color-text;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-code {
+      margin: 4px 0 0;
+      font-size: 10px;
+      color: $secondary-color-text;
+      font-weight: 400;
+    }
+
+    &-price {
+      margin: 24px 0 0;
+      font-size: 12px;
+      color: $secondary-color-text;
+      font-weight: 400;
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+
+      span {
+        font-weight: 700;
+        font-size: 16px;
+        color: $primary-color-text;
```

Comment:
The variable `$primary-color-text` is used here but not defined in this file. Ensure it is defined or imported from another SCSS file to avoid compilation errors.